homeassistant/components/sighthound/image_processing.py

```diff
@@ -81,6 +93,8 @@ def process_image(self, image):
         self._image_height = metadata["image_height"]
         for person in people:
             self.fire_person_detected_event(person)
+        if hasattr(self, "_save_file_folder") and self._state > 0:
```

Comment:
Don't check with `hasattr`. Check for `None`.

Reply:
ok